astropy/cosmology/parameter/tests/test_parameter.py

```diff
@@ -265,12 +259,14 @@ class TestParameter(ParameterTestMixin):
     """
 
     def setup_class(self):
+        Param = Parameter(
+            doc="Description of example parameter.",
+            unit=u.m,
+            equivalencies=u.mass_energy(),
+        )
+
         class Example1(Cosmology):
-            param = Parameter(
```

Comment:
I think this became a bit less clear

Reply:
maybe a little here, but it made line 287 clearer. `Param` is attached to both `Example1` and `Example2`, but with slightly different requirements.